libs/ktem/ktem/reasoning/prompt_optimization/rewrite_question.py

```diff
@@ -0,0 +1,37 @@
+from ktem.llms.manager import llms
+
+from kotaemon.base import BaseComponent, Document, HumanMessage, Node, SystemMessage
+from kotaemon.llms import ChatLLM, PromptTemplate
+
+DEFAULT_REWRITE_PROMPT = (
+    "Given the following question, rephrase and expand it "
+    "to help you do better answering. Maintain all information "
+    "in the original question. Keep the question as concise as possible. "
+    "Give answer in {lang}\n"
+    "Original question: {question}\n"
+    "Rephrased question: "
+)
+
+
+class RewriteQuestionPipeline(BaseComponent):
```

Comment:
Please rename this to `SimpleRewriteQuestionPipeline` or something similar to better differentiate it from its few-shot counterpart.